### ISSUE TITLE
Three dots menu is hidden

### ISSUE THREAD

**Issue by seandop** [maykar/custom-header#207]
Since 1.4.7 release, the three dot menu on Lovelace is no longer visible. On page load, it does appear, but is then covered up and inaccessible. Thus, I'm unable to reconfigure Custom Header. Reverting to version 1.4.6 solves the issue.

**Comment by seandop** [comment]
Updated info above.

**Comment by seandop** [comment]
> 
> 
> In order for me to recreate and troubleshoot your issue, please read the bot's previous reply. Follow the post's steps and provide the info requested. This issue will remain closed until more information is provided.

Updated info above. @maykar

**Comment by maykar** [comment]
`<br>` is the HTML equivalent to hitting return. It is an empty line break.

By using that `button_text` config you've told CH to replace the options icon with an empty line break.

TL;DR
Remove this:
```
button_text:
    options: <br>
```

**Comment by seandop** [comment]
> 
> 
> `<br>` is the HTML equivalent to hitting return. It is an empty line break.
> 
> By using that `button_text` config, you've told CH to replace the options icon with an empty line break and nothing else.
> 
> TL;DR
> Remove this:
> 
> ```
> button_text:
>     options: <br>
> ```

Well, I feel dumb. I hadn't noticed that -- I think it was added by one of my themes and it never occurred to me. Confirmed that this fixes my issue with 1.5.0 (not sure why 1.4.6 and before seemed to ignore this?). Thanks

**Comment by maykar** [comment]
Seems like there was a bug in 1.4.6 that has since been resolved, funny that it lined up with your issue and "fixed" it.
